src/structurefinder/PyBernyOpt.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+from nwchemex import chemist
+from berny import Berny, geomlib, optimize
+from berny.solvers import MopacSolver
+
+# Molecule for testing
+#h2 = chemist.Molecule()
+#a1       = chemist.Atom("H", 1, 1.0079, 0.0, 0.0, 0.0)
+#a2       = chemist.Atom("H", 1, 1.0079, 0.0, 0.0, 1.0)
+#h2.push_back(a1)
+#h2.push_back(a2)
+
+def optimize_pyberny(molecule):
+    """
+    This function takes a chemical system and uses PyBerny
+    to optimize the provided geometry (molecule) and 
+    returns the energy of the optimized system.
+    """
+    # Convert a Chemical System to an XYZ coordinate string
+    xyz = str(molecule.size()) + "\n\n" + str(molecule.nuclei)
```

Comment:
I wouldn't rely on the string conversion to get this right; rather I'd manually loop over the nuclei in in `molecule.nuclei` and print them out in the format you want. Take a look [here](https://github.com/NWChemEx/Chemist/blob/master/tests/python/unit_tests/chemical_system/nucleus/test_nuclei.py) for examples of using the `Nuclei` class from Python.